database/data generation/psql.sql

```diff
@@ -0,0 +1,16 @@
+-- psql.sql
+
+DROP DATABASE IF EXISTS photos;
+
+CREATE DATABASE photos;
+
+-- make sure using photos database
+\c photos;
+
+-- Create items table
+CREATE TABLE listings (
+  listing_id INT,
+  url VARCHAR,
```

Comment:
url is a reserved keyword in postgres, might be causing problems? not sure but I think I remember someone saying to use something different.